common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -973,29 +974,33 @@ def get_transcript_from_contentstore(video, language, output_format, youtube_id=
     return transcript_content, transcript_name, Transcript.mime_types[output_format]
 
 
-def get_transcript(course_id, block_id, lang=None, output_format=Transcript.SRT, is_bumper=False):
+def get_transcript(video, lang=None, output_format=Transcript.SRT, youtube_id=None):
     """
     Get video transcript from edx-val or content store.
 
     Arguments:
-        course_id (CourseLocator): course identifier
-        block_id (unicode): a unique identifier for an item in modulestore
+        video (Video Descriptor): Video Descriptor
         lang (unicode): transcript language
         output_format (unicode): transcript output format
-        is_bumper (bool): indicates bumper video
+        youtube_id (unicode): youtube video id
 
     Returns:
         tuple containing content, filename, mimetype
     """
-    usage_key = BlockUsageLocator(course_id, block_type='video', block_id=block_id)
-    video_descriptor = modulestore().get_item(usage_key)
+    transcripts_info = video.get_transcripts_info(include_val_transcripts=True)
```

Comment:
haven't we removed `include_val_transcripts=True`?

Reply:
this is removed